commercetools/client_queryinput.go

```diff
@@ -39,6 +39,9 @@ type QueryInput struct {
 
 	Limit  int
 	Offset int
+
+	// Extra query arguments.
+	Extra url.Values
```

Comment:
Just leaving a comment here for later. I would prefer a more rigid approach to extra query arguments, so similar to `Limit` and `Offset` just having a struct field for any additional params such as `staged`.

But as we are regenerating everything soon anyway (https://github.com/labd/commercetools-go-sdk/pull/63#issuecomment-818827221 ) I am ok with merging for now